.pre-commit-config.yaml

```diff
@@ -10,22 +10,25 @@ repos:
             )$
 
 -   repo: https://github.com/pre-commit/mirrors-mypy
-    rev: v0.761
+    rev: v0.910
```

Comment:
Had compatibility issues with Python 3.9 -- this new version seems to be working much better with improved error handling and warnings.